OSDOCS-9286 AWS: Support for AWS Outposts (GA) (#71793)
openshift/openshift-docs: installing/installing_aws/installing-aws-outposts.adoc

Reviewer: Please remove all hard wrapping in this file.

:_mod-docs-content-type: ASSEMBLY
[id="installing-aws-outposts"]
= Installing a cluster on AWS with compute nodes on AWS Outposts
include::_attributes/common-attributes.adoc[]
:context: installing-aws-outposts

toc::[]

In {product-title} version 4.14, you could install a cluster on
Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview.

> As of {product-title} version {product-version}, this installation method is no longer supported.

Reviewer: Should the `product-version}` variable be replaced with the first version that no longer supports this feature? Technically the phrase "As of" implies we're talking about the first version that dropped support, not whether it's supported in the current version a user is looking at docs for.

```suggestion
As of {product-title} version 4.15, this installation method is no longer supported.
```

> Instead, you can

Reviewer: Totally up to you but if the alternative is at all important to highlight, making it its own paragraph makes it easier to see and makes the above paragraph smaller/more scannable. But this is bikeshedding so feel free to ignore.
```suggestion

Instead, you can
```

xref:../../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[install a cluster on AWS into an existing VPC]

> and provision compute nodes on AWS Outposts as a postinstallation configuration task.

Reviewer 2: ```suggestion
and provision compute nodes on AWS Outposts as a post installation configuration task.
```

Author: I've got it as "postinstallation" because that's how we refer to it in the main doc: https://docs.openshift.com/container-platform/4.15/post_installation_configuration/index.html

<img width="1049" alt="Screenshot 2024-02-21 at 9 56 44 PM" src="https://github.com/openshift/openshift-docs/assets/93731574/8006305e-7694-4efb-a016-aa67fad734b5">

Reviewer 2: I was wondering about it too. SGTM :)

Reviewer 3: I don't think anyone likes it, but that's it the style guide :sweat_smile:

> //For more information, see PUT AN XREF HERE SOON

Reviewer 2: sgtm. Will we wait for the PR  https://github.com/openshift/openshift-docs/pull/71263 to merge before this?

Author: Yes, that's where the link to Jeana's content will go :)

Reviewer 3: https://github.com/openshift/openshift-docs/pull/71263/files
